spec/controllers/api_response/login_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+RSpec.describe LoginController, type: :request do
+
+    context "login successfully" do
+    before {Admin.create!(name:'test', mobile:'123', password:'123')}
+        it {
```

Comment:
use `do ... end` instead of `{... }`

Reply:
removed { . . } block, and replaced with do-end block